### ISSUE TITLE
download speeds can be _very_ slow in BCE

### ISSUE THREAD

**Issue by paciorek** [ucberkeley/bce#61]
I've run into this earlier this summer as well in spring-2015...

Downloading a test 150 Mb file using wget in BCE downloads at ~50 KB/s when the host is a Linux machine. Natively downloading on the host Linux machine it is 100 MB/s. Similarly in BCE on a Mac laptop, downloading is ~50 KB/s.

**Comment by davclark** [comment]
I'm wondering if this is a network or rather a filesystem issue. I've had issues in linux in general when, for example, writing to a USB key. Does your filesystem otherwise perform close to native speed?

**Comment by paciorek** [comment]
I think the filesystem is fine - I just copied a big-ish file from one
place to another from within the BCE filesystem and didn't notice anything
awry. If anyone has suggestions of how to test filesystem performance in a
standard way, let me know.

I'll note that using wget or scp, the download speeds start at a high rate
(say 10 MB/s or 500 KB/s) but then quickly drop to like 30 KB/s.

**Comment by aculich** [comment]
Are shared folders involved? If so, try removing all shared folders and check the download speeds. The shared folders are known to have very slow performance (among other problems).

See also: [Comparing Filesystem Performance in Virtual Machines](http://mitchellh.com/comparing-filesystem-performance-in-virtual-machines)

**Comment by paciorek** [comment]
No, I haven't enabled shared folders in the VM where I am seeing the slow
downloading.
